app/assets/javascripts/discourse/app/templates/topic.hbs

```diff
@@ -300,7 +300,7 @@
                   removeTopicTimer=(action "removeTopicTimer" model.topic_timer.status_type "topic_timer")}}
 
               {{#if showSelectedPostsAtBottom}}
```

Comment:
To be clear, the bottom part shows the selected-posts interface if `showSelectedPostsAtBottom` is true. This code hides it when that's the case, because otherwise you see the interface twice. 

This block is used by non-mobile interfaces I think. 

I would prefer to only have one but this seemed like the fix least likely to break stuff.